legend-engine-xts-snowflake/legend-engine-xt-snowflake-pure-test/src/main/resources/core_snowflake_test/core_snowflakeapp/showcase/showcaseElements.txt

```diff
@@ -0,0 +1,205 @@
+
```

Comment:
We need to standardize these, and a file type we been agreeing on is to use `.legend`.

Reply:
A subsequent PR will change this